File: Android_Final/app/src/main/java/com/example/android_etpj/models/FeedbackQuestion.java
package com.example.android_etpj.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class FeedbackQuestion implements Serializable {
    @SerializedName("Feedback")
    private Feedback feedback;

    @SerializedName("Questions")
    private List<Integer> questions;

    public FeedbackQuestion() {
    }

    public Feedback getFeedback() {
        return feedback;
    }

    public void setFeedback(Feedback feedback) {
        this.feedback = feedback;
    }

    public List<Integer> getQuestions() {
        return questions;
    }

    public void setQuestions(List<Integer> questions) {
        this.questions = questions;
    }
}

File: API/API/ViewModel/FeedbackQuestion.cs
using API.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace API.ViewModel
{
    public class FeedbackQuestion
    {
        public Feedback Feedback { get; set; }
        public List<int> Questions { get; set; }
    }
}